rethinking-css.md

```diff
@@ -1,24 +1,24 @@
 # Rethinking CSS
 
-In this article, I hope to define what I feel are the biggest issues with CSS, and ultimately, how those issues can be overcome.
+In this article I hope to define what I feel are the biggest issues with CSS and, ultimately, how those issues can be overcome.
```

Comment:
Revert comma removal after "article". There's a natural pause that should be communicated